_posts/2016-04-27-discounted-goruco-tickets-for-underrepresented-people.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+excerpt_separator: <!--more-->
+---
+
+<p>
+  This year we are offering discounted tickets for those who self-identify
+  as underrepresented in technology via a discount code.</p>
+
+<h3>Why offer a discount at all?</h3>
+
+<p>
+  GORUCO's organizing committee believes our event
+  should reflect and model the values of our communities.
+  Diversity and inclusivity have rightly risen to prominence
+  as major points of concern in both our local community and
+  the broader Ruby community.</p>
+
+<!--more-->
+
+<h3>How to?</h3>
+
+<p>
+  The discount code is: "diversity."
+  You can enter it on this page as you buy your ticket.
+  It entitles the purchaser to a $100 discount in ticket price.</p>
```

Comment:
This is rendering just as a paragraph (not sure if that was the intent) but since this is a blog post, not the ticketing page, the use of the word "this" is a bit confusing. Suggest changing to:

_The discount code is "<em>diversity</em>" and it will save you $100 off of a full-priced ticket. You can enter it on the page as you buy your ticket._